index.html

```diff
@@ -314,7 +315,12 @@ <h2>Contact</h2>
           Stop wasting time and money designing and managing a website that
           doesn't get results. Happiness guaranteed!
         </p>
-        <img src="assets/img/map.png" alt="map" />
+        <div class="map-container" >
+          <div style="width: 100%"><iframe scrolling="no" marginheight="0" marginwidth="0"
```

Comment:
I think you can delete  style="width: 100%", because you add it in the css already

Reply:
yes can delete that whole `<div style="width: 100%">` now I believe as well

Reply:
yep you're right, should be good now